CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -1,3 +1,24 @@
+# v1.11.7 - Changelog since v1.11.6
+
+## Changes by Kind
+
+### Uncategorized
+
+- Properly wrap error from GCE Images.Get() API call, to fix a potential nil-ptr dereference ([#1516](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1516), [@k8s-infra-cherrypick-robot](https://github.com/k8s-infra-cherrypick-robot))
```

Comment:
#1516 is technically on 1.11.6. I would either move it to 1.11.6, or change the subtitle above to "Changelog since v1.11.5"

Reply:
fixed